packages/engine/src/worker/mod.rs

```diff
@@ -127,9 +127,9 @@ impl WorkerController {
 
     async fn _run(&mut self) -> Result<()> {
         // TODO: Rust, JS
-        let mut js_handle = self.js.run().await?;
+
         // let mut py_handle = self.py.run().await?;
-        // let mut rs_handle = self.rs.run().await?;
+        let mut js_handle = self.js.run().await?;
```

Comment:
I think you've removed
```rust
// let mut rs_handle = self.rs.run().await?;
```
and the await below.

If we're leaving the commented out py_handle then I don't see why we're removing the rs one

Reply:
Resolving, has been re-added